index.js

```diff
@@ -77,6 +77,10 @@ function sanitizeHtml(html, options, _recursing) {
       allowedAttributesMap[tag].push('class');
     }
 
+    classes.map(function(clss) {
+      return quoteRegexp(clss).replace(/\\\*/g, '.*')
```

Comment:
Agreed. Removing this line, the tests still pass. I'm not sure what it's attempting to do here.